libraries/AP_HAL_ChibiOS/hwdef/uav-dev-fc-um982/defaults.parm

```diff
@@ -0,0 +1,24 @@
+CAN_P1_DRIVER 1
+CAN_P2_DRIVER 2
+
+BATT_MONITOR 8
+
+GPS1_TYPE 25
+
+NET_ENABLE 1
+NET_OPTIONS 0
+NET_DHCP 0
+NET_P1_TYPE 2
+NET_P1_PROTOCOL 2
+NET_P1_PORT 14550
+
+NET_P2_TYPE 4
+NET_P2_PROTOCOL 2
+NET_P2_PORT 5600
+
+SERIAL2_PROTOCOL 5
```

Comment:
All of these port protoocols should be moved to the hwdef to save memory..

Reply:
done